ROADMAP.md

```diff
@@ -8,21 +8,25 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.9
+## Rook Ceph 1.10
 
-The following high level features are targeted for Rook v1.9 (early April 2022). For more detailed project tracking see the [v1.9 board](https://github.com/rook/rook/projects/24).
+The following high level features are targeted for Rook v1.10 (early August 2022). For more detailed project tracking see the [v1.10 board](https://github.com/rook/rook/projects/26).
+
+* Remove support for Ceph Octopus (support remains for Pacific and Quincy) [#10338](https://github.com/rook/rook/issues/10338)
```

Comment:
Even with the documentation restructure/ rework being partially done today, should we add it as we are still working on further rewrites of the documentation?

Reply:
The roadmap is really for capturing changes in future releases. Anything we are going to backport to the current release I don't think needs to be captured here. Anytime we update the docs, we're likely going to backport it though unless it's a major restructure. But maybe I'm not aware of all the plans you have for the docs, so we should chat more. :)